src/services/copyConfig/rba/policy.js

```diff
@@ -17,18 +17,22 @@ export default class Policy {
 
   async get() {
     const url = UrlBuilder.buildUrl(Policy.#NAMESPACE, this.originDataCenter, 'accounts.rba.getPolicy', this.#credentials.gigyaConsole)
```

Comment:
'accounts.rba.getPolicy' the endpoint is hard coded on the parameter, consider creating a variable with the endpoint like:

static #GETPOLICY = 'accounts.rba.getPolicy'